src/main/java/be/syntra/devshop/DevshopBack/entities/Cart.java

```diff
@@ -0,0 +1,64 @@
+package be.syntra.devshop.DevshopBack.entities;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.*;
+import org.hibernate.annotations.Cascade;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Cart {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "cart_id")
+    private Long id;
+
+
+    @OneToOne(targetEntity = User.class, mappedBy = "activeCart")
```

Comment:
try to be consistent. activeCart -> active_cart

Reply:
please change all the names with camelcasing.

Reply:
mappedBy = "activeCart" refers to private Cart activeCart in the User class.
If you would make it active_cart it wil give an error.

![error](https://user-images.githubusercontent.com/50916349/77230818-462fe880-6b97-11ea-9621-724380ed6e31.PNG)

Reply:
ah you placed comment above the line in question